### PR TITLE
web-angular: Ensure compatibility between all angular packages

### PR DESCRIPTION
**Pull request by kevmoo**

### CODE REVIEW

File: templates/web-angular/pubspec.yaml [hunk 1]
```diff
@@ -8,11 +8,14 @@ environment:
...
+  angular: 5.0.0-alpha+11
+  angular_components: 0.9.0-alpha+11
+
+  # Needed to ensure 0.4.0-alpha+12 (or greater) is not accessed
+  angular_compiler: 0.4.0-alpha+11
 
 dev_dependencies:
-  angular_test: ^2.0.0-alpha+9
+  angular_test: 2.0.0-alpha+9
   build_runner: ^0.8.7
```
[the remark this answers is not in the document]
Comment (chalin) [reply templates/web-angular/pubspec.yaml]: (Ideally with a comment containing a link to the issue we'll use to track the source of the problem.)

**Comment by kevmoo** [comment]
PTAL

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,7 +1,9 @@
 ## 2.0.1
 
-- Updated the dependency on `package:build_web_compilers` in all `web-`
-  templates.
+- All `web-` templates:
+  - Updated the dependency on `package:build_web_compilers`.
+- `web-angular` template:
+  - Update all dependencies on `anuglar` packages to ensure compatibility.
```
Comment (kwalrath) [inline CHANGELOG.md]: Update -> Updated
anuglar -> angular

**Comment by kwalrath** [comment]
LGTM once those typos are fixed. (I can do that, if you're busy.)

File: CHANGELOG.md (hunk 2 above)
Comment (kwalrath) [inline CHANGELOG.md]: Oops, we changed the pubspec, so we should change this. It's only angular_compiler that changed.

**Pull request merged**